tests/src/Transformation/Transformer/ArgumentTransformer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @file
+ * Contains Drupal\nexteuropa\Transformation\Transformer\ArgumentTransformer.
+ */
+
+namespace Drupal\nexteuropa\Transformation\Transformer;
+
+use Behat\Behat\Definition\Call\DefinitionCall;
+use Behat\Behat\Transformation\Transformer\ArgumentTransformer as ArgumentTransformerInterface;
+
+/**
+ * Class ArgumentTransformer in charge of the "nept_element" transformation.
+ *
+ * @package Drupal\nexteuropa\Transformation\Transformer
+ */
+class ArgumentTransformer implements ArgumentTransformerInterface {
+  /**
+   * Extension configuration.
+   *
+   * @var
+   */
+  private $config;
+
+  /**
+   * ExtensionTransformer constructor.
+   */
+  public function __construct($config) {
+    $this->config = $config;
+  }
+
+  /**
+   * Implements supportsDefinitionAndArgument().
+   *
+   * @inheritDoc
+   */
```

Comment:
I will change but do not use a Drupal reference in project that does not concern Drupal directly.